backend/main.py

```
@@ -0,0 +1,190 @@
+@app.post("/auth/login", response_model=schemas.Token)
+def login(form_data: OAuth2PasswordRequestForm = Depends()):
+    # パスワードチェックは省略
+    return {
+        "access_token": "fake-jwt-token-for-hackathon",
+        "token_type": "bearer"
+    }
+
+
+# Spots
+@app.get("/spots", response_model=List[schemas.SpotResponse])
+def get_spots(lat: Optional[float] = None, lng: Optional[float] = None, radius: Optional[float] = None):
```

Ruff (0.14.7)

Unused function argument: `lat`

(ARG001)